esp/src/src-react/components/SourceFiles.tsx

```diff
@@ -49,16 +37,16 @@ export const SourceFiles: React.FunctionComponent<SourceFilesProps> = ({
                 width: 27,
                 selectorType: "checkbox"
             }),
-            Name: tree({
```

Comment:
The Dojo version will display folders as an expandable tree. Will this just be a flat list instead?